spec/support/features/doc_auth_helper.rb

```diff
@@ -35,6 +35,18 @@ def click_doc_auth_back_link
     click_on '‹ ' + t('forms.buttons.back')
   end
 
+  def idv_ip_get_started_step
+    idv_inherited_proofing_step_path(step: :get_started)
+  end
+
+  def idv_ip_verify_info_step
+    idv_inherited_proofing_step_path(step: :verify_info)
```

Comment:
I know we talked about this... seems like we should have our own helper file.  I don't like that we're referencing doc_auth, when we are not using doc_auth.  Can always refactor later... if we have to.

Reply:
Yep, plan to refactor later. We will be reusing some of the methods from this class for final verfication steps after lexis nexis calls. ( Not sure if we will add another new file to place common code between doc_auth / ip)

Reply:
I think moving these into their own helper file may actually be helpful. It does not prevent you from using any of the methods in here. Especially if you include this help in that one.

Reply:
We will do that Jonathan.